cea/demand/thermal_loads.py

```diff
@@ -328,3 +322,32 @@ def update_timestep_data_no_conditioned_area(tsd):
     return tsd
 
 
+def get_hours(bpr):
+    """
+
```

Comment:
so... what does this function do? where is it used?

Reply:
yes. still adding to documentation.